### PR TITLE
license test for IBM

### PR DESCRIPTION
**Pull request by Balachandar-Pan**
**What this PR does / why we need it**:
license test for IBM
**Which issue(s) this PR fixes** (optional)
Closes #PTX-18543

**Special notes for your reviewer**:
https://jenkins.pwx.dev.purestorage.com/job/Torpedo/job/tp-nextpx-iks-catalog-pl/165/

### CODE REVIEW

File: tests/basic/license_test.go [hunk 1]
```diff
@@ -715,6 +748,55 @@ var _ = Describe("{DisableCallHomeTest}", func() {
...
+			log.InfoD("%v", summary.SKU)
+			strmatch := strings.Contains(summary.SKU, ibmTestLicenseSKU)
+			dash.VerifyFatal(strmatch, true, "IBM test license got activated?")
+
+			Step("Compare PX-IBM-Test License features vs activated license", func() {
+				log.InfoD("Compare with IBM cloud test license features")
+				for _, feature := range summary.Features {
+					// if the feature limit exists in the hardcoded license limits we test it.
+					if _, ok := ibmLicense[Label(feature.Name)]; ok {
+						Expect(feature.Quantity).To(Equal(ibmLicense[Label(feature.Name)]),
```
Comment (lsrinivas-pure) [inline tests/basic/license_test.go:786]: use dash.verifyFatal here

File: tests/basic/license_test.go [hunk 2]
```diff
@@ -715,6 +748,55 @@ var _ = Describe("{DisableCallHomeTest}", func() {
...
+	stepLog := "Get SKU and compare with IBM cloud test license"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("license-validation-%d", i))...)
+		}
+
+		ValidateApplications(contexts)
```
Comment (lsrinivas-pure) [inline tests/basic/license_test.go:767]: do u need to deploy apps for this valdiation?

File: tests/basic/license_test.go (hunk 1 above)
Comment (lsrinivas-pure) [inline tests/basic/license_test.go:786]: is this comparison correct quantity to ibmLicense[Label(feature.Name)]?

**Review by lsrinivas-pure — CHANGES_REQUESTED** [review]

**Comment by Balachandar-Pan** [comment]
adding code to different branch

**Pull request closed without merging**